pkg/controller/podgroup_test.go

```diff
@@ -447,6 +519,39 @@ func TestCalculatePGMinResources(t *testing.T) {
 				corev1.ResourceMemory: resource.MustParse("65Gi"),
 			},
 		},
+		"without worker without priorityClass": {
+			minMember: 3,
+			job: &kubeflow.MPIJob{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test",
+				},
+				Spec: kubeflow.MPIJobSpec{
+					MPIReplicaSpecs: map[kubeflow.MPIReplicaType]*kubeflow.ReplicaSpec{
+						kubeflow.MPIReplicaTypeLauncher: {
+							Replicas: ptr.To[int32](1),
```

Comment:
```suggestion
							Replicas: ptr.To[int32](3),
```

This should be `3` since `minMember` is 3.

Reply:
We can only have one launcher replica, so I set minMember to 1 to be more consistent. Lmk if that works please